ethers-contract/src/multicall.rs

```diff
@@ -0,0 +1,204 @@
+extern crate hex;
+
+use ethers_core::{
+    abi::{Detokenize, Function, Token},
+    types::{Address, BlockNumber, NameOrAddress, TransactionRequest, U256},
+};
+use ethers_providers::JsonRpcClient;
+use ethers_signers::{Client, Signer};
+use hex::FromHex;
+use lazy_static::lazy_static;
+
+use std::{collections::HashMap, sync::Arc};
+
+use crate::call::{ContractCall, ContractError};
+use crate::multicall_contract::MulticallContract;
+
+lazy_static! {
```

Comment:
I prefer using `OnceCell` instead, syntax is fairly similar:https://docs.rs/once_cell/1.4.0/once_cell/sync/struct.Lazy.html